modules/react/popup/lib/PopupCard.tsx

```diff
@@ -87,7 +88,7 @@ export const PopupCard = createComponent('div')({
         padding={padding}
         {...props}
       >
-        <Card.Body>{children}</Card.Body>
+        {children}
```

Comment:
This removes the extra wrapper `div` in Popups and instead manually brings in the `type.levels.subtext.large` typing to the `<StyledPopupCard>` styled component. This allows better style overrides.

Reply:
Already done